packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -94,6 +94,11 @@
                 "type": "string",
                 "description": "%sfdx_project_text%",
                 "default": "${workspaceRoot}"
+              },
+              "trace": {
+                "type": ["boolean", "string"],
```

Comment:
Could you elaborate on what `"type": ["boolean", "string"]` means? Why do we want to take two kinds of distinct types?

Reply:
you can set `trace: true`, which is equal/shortcut to `trace: 'all'`

Reply:
👍 but let's change the type from 'va', 'dap' to full words.

Reply:
done